### ISSUE TITLE
Thumbnails for tree preview - generation s3 bucket

### ISSUE THREAD

**Issue by petterjv** [pimcore/pimcore#5396]
# Bug Report

### Expected behavior
There is an issue with generating tree preview thumbnails when using custom stream such as Amazon s3. 

### Actual behavior
Tree preview thumbnail can be generated with custom stream.

### Steps to reproduce
Setup Pimcore for S3 bucket storage according to documentation. Clear temporary files.

Run the command:
bin/console pimcore:image:low-quality-preview

Output:
In Image.php line 208:                                                                                                
  insufficient image data in file `/tmp/magick-53GAkO63mDDU4s' @ error/jpeg.c/ReadJPEGImage/1183  

I'm not sure how to fix it in a correct way, so I cannot provide a PR.

### Code example
```php
// fallback
if (class_exists('Imagick')) {
    // Imagick fallback
    $path = $this->getThumbnail(Image\Thumbnail\Config::getPreviewConfig())->getFileSystemPath();
    $imagick = new \Imagick($path);
//...
```

**Comment by petterjv** [comment]
We have temporary fixed it with:
```
pimcore:
    assets:
        image:
            low_quality_image_preview:
                enabled: false

**Comment by brusch** [comment]
This has been fixed by #8562
